### PR TITLE
Oiels additional information

### PR DESCRIPTION
**Pull request by Tllew**
### Aim

[LTD-](https://uktrade.atlassian.net/browse/LTD-)

### CODE REVIEW

File: core/builtins/custom_tags.py [hunk 1]
```diff
@@ -756,15 +756,18 @@ def task_list_additional_information_status(data):
     """
     Returns 'not started' if length of data given is none, else returns 'done'
     """
-    if all([data.get(field) in ["", None] for field in F680.REQUIRED_FIELDS]):
+    case_map = {"f680": F680, "oiel": OIEL}
+    case_type = data["case_type"]["reference"]["key"]
+    licence_type = case_map[case_type]
```
Comment (currycoder) [inline core/builtins/custom_tags.py:761]: Will "siel" give a `KeyError`?

Comment (Tllew) [reply core/builtins/custom_tags.py:761]: `					{% if application.case_type.reference.key|application_type_in_list:"ADDITIONAL_INFORMATION" %}
`

this should negate that, I can change it to a .get though

File: exporter/applications/constants.py [hunk 2]
```diff
@@ -33,6 +33,18 @@ class F680:
...
 
+class OIEL:
+    FIELDS = [
+        "nature_of_product",
+        "number_of_siels_last_year",
+        "siels_issued_last_year",
+        "destination_countries",
+        "purely_commercial",
+    ]
+    REQUIRED_FIELDS = ["nature_of_product", "siels_issued_last_year", "purely_commercial"]
```
Comment (saruniitr) [inline exporter/applications/constants.py:44]: `nature_of_products`

**Review by saruniitr — APPROVED** [review]

File: exporter/applications/constants.py (hunk 2 above)
Comment (Tllew) [reply exporter/applications/constants.py]: updated

**Review by currycoder — APPROVED** [review]

**Pull request merged**